web-console/4.0-designs/create-from-YAML/create-from-YAML.md

```diff
@@ -0,0 +1,32 @@
+# Create from YAML
+
+A Create from YAML button will be added to the project selector bar, making it accessible from every page within the console.
+
+
+**Create from YAML button**
+![button](img/YAML-Button.png)
+
+* The Create from YAML button will be in the project selector bar
```

Comment:
Good points on the project stuff. I can definitely see a use-case for a file setting up a namespace and then creating objects within that namespace. I have one of these for setting up our Operator certification cluster. It makes a namespace per partner, creates cluster Roles and assigns RoleBindings into each namespace.

Aside from where this trigger goes, what you do think of reacting to any YAML dragged into the Console window if you weren't already in the middle of editing something? If you dropped it, it would redirect you to this page without a project bar.

What about putting it in the bottom of the nav bar? Sticks to the bottom of the window if the nav is short and is pushed down if its longer?

Reply:
@robszumski this sounds like an interesting capability.  Would this be post 4.0?  Wonder if we could prototype and test it before we decide it's something we want to do?

Reply:
@robszumski @spadgett just thinking about the 2 possible options at this time ... 
- placed on the Project Selector bar, which is not always present
- placed last navigation section, which is not always viewable

I'm wondering if it makes sense to have it in both places, and if clicked from the Project Selector bar,  include:
metadata:
  namespace: project-name

Does that make any sense 🤔

Reply:
The other possibility is to place it on the catalog page.

Reply:
@serenamarie125 Assuming you are talking about the drag/drop, I imagine we could do this whenever as the building blocks are there. We'd just need the multi-object support in the editor.

> I'm wondering if it makes sense to have it in both places

I'm hesitant to do this...I think we can find one place that makes sense. For most users, the project bar will be visible most of the time, which makes me lean towards that.